packages/grid/x-data-grid-pro/src/components/DataGridProVirtualScroller.tsx

```diff
@@ -307,9 +307,9 @@ const DataGridProVirtualScroller = React.forwardRef<
 
   // Create a lookup for faster check if the row is expanded
   const expandedRowIdsLookup = React.useMemo(() => {
```

Comment:
I just noticed that we might reuse this lookup here as well:
https://github.com/mui/mui-x/blob/d7ac79c5ae5060e793dc723bbd06c8f4b0f4f1bc/packages/grid/x-data-grid-pro/src/components/DataGridProVirtualScroller.tsx#L241
This should be a bit more efficient.
Would you like to include this change in this PR?

Reply:
@cherniavskii Absolutely, will do!